Source/NSApplication.m

```diff
@@ -2458,6 +2459,14 @@ - (NSWindow*) iconWindow
   return _app_icon_window;
 }
 
+- (NSDockTile *) dockTile
+{
+  if (!_dock_tile)
+    {
+      _dock_tile = [[NSDockTile alloc] init];
```

Comment:
This needs to be released in the `dealloc`method.

Reply:
fixed